CHANGELOG.md

```diff
@@ -1,5 +1,154 @@
 ### [Versions](https://material-ui.com/versions/)
 
+## 5.0.0-beta.5
+
+<!-- generated comparing v5.0.0-beta.4..next -->
+
+_Aug 24, 2021_
+
+Big thanks to the 26 contributors who made this release possible. Here are some highlights ✨:
```

Comment:
```suggestion
A big thanks to the 26 contributors who made this release possible. Here are some highlights ✨:
```

Reply:
We should change the template for `release:changelog` with this change :)